Taskfile.yml

```diff
@@ -78,13 +78,12 @@ tasks:
     cmds:
       - yarn wsrun --stages --report --fast-exit --exclude-missing ws:clean
 
-  spec:build:
-    dir: workspaces/ui
+  spec_viewer:build:
```

Comment:
Should I be... nuking the `ui` building code here?

Reply:
on our call we discussed changing the webpack paths.js to also parameterize the output directory from the webpack build. in this previous incarnation, it was used to build them in succession into the same output directory, so it needed to be cleaned in between

Reply:
Sorry, I meant: these changes remove support for building the existing UI. Is that what I should be doing?

Reply:
oh sorry. so there were previously 3 ui builds: the one used by the local-cli, the one used for the shared specs viewer, and the one used by the demo ui. This workflow only handles the shared specs and demo builds. The build used by local-cli was actually produced in the ui/package.json just before publishing, so that's not affected by these changes. (However, currently ui-v2 is not running that build, but that's a separate issue you don't need to worry about)

Reply:
until we allocate resources for getting parity with the demo build, we should just leave the deployed one running and not do anything with ui-v2

Reply:
Rebasing ontop of `swarm-1` resolves this: `ui` is gone